Cargo.toml

```
@@ -3,6 +3,22 @@ name = "rmathlib"
 description = "A Rust Port of R's C Library of Special Functions"
 version = "1.0.0"
 edition = "2021"
+authors = [
+  "Rik Huijzer <rik@huijzer.xyz>",
+  "Jose Storopoli <jose@storopoli.io>",
+]
+license = "GPL 3.0"
+categories = ["science"]
```

```suggestion
categories = ["science", "mathematics", "algorithms"]
```
mathematics is obvious. Algorithms maybe too?